app/controllers/groups_controller.rb

```diff
@@ -36,6 +36,7 @@ class GroupsController < ApplicationController
       groups.where(automatic: true)
     }
   }
+  ADD_MEMBERS_LIMIT = 1000
```

Comment:
To prevent warnings when reloading it's recommended to use the `||=` operator here.